packages/experimental/mosaic/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import faker from 'faker';
+import { Buildings, DotsThree, Envelope, User } from 'phosphor-react';
+import React from 'react';
+
+// TODO(burdon): Add note to @dxos/react-components/plugin
+// storybook config (files and modules)
+// dxosTheme import
+
+import { Button, DensityProvider, getSize } from '@dxos/react-components';
+import '@dxosTheme';
+
+import { Card } from './Card';
+import { CardActions } from './CardActions';
+import { CardContent } from './CardContent';
+import { CardHeader } from './CardHeader';
+
+// TODO(burdon): Copy storybook structure from @dxos/react-components.
+
+// TODO(burdon): IconButton.
+// TODO(burdon): Density: Input, Checkbox.
+// TODO(burdon): ListItem as CardHeader.
+
+// TODO(burdon): Column on mobile; no rounded corner if mobile column.
+
+// TODO(burdon): Factor out default semantic colors.
+
+const styles = {
+  header: 'bg-blue-300 dark:bg-zinc-800'
+};
+
+// TODO(burdon): Test embedded table/lists (e.g., checkbox in side gutter).
+
+const Test = () => {
+  return (
+    <div className='flex flex-col w-full md:w-[390px] m-4 space-y-8'>
+      <DensityProvider density='fine'>
+        <Card slots={{ root: { className: 'text-sm' } }}>
```

Comment:
To your point about aiming to surface an API that minimizes the need for style interventions, perhaps the story here should demonstrate what Cards look like with zero intervention.